src/steps/writing/creation/dotVSCode.ts

```diff
@@ -50,5 +60,17 @@ export async function createDotVSCode(options: Options) {
 			"eslint.rules.customizations": [{ rule: "*", severity: "warn" }],
 			"typescript.tsdk": "node_modules/typescript/lib",
 		}),
+		"tasks.json": await formatJson({
```

Comment:
[Question] I don't personally like using VS Code's built-in tasks feature. I personally find it convoluted to have background tasks, builders, etc. I like having a `tsc -w` in a terminal prompt along with my other prompts. Does a `build` task _need_ to exist? Thoughts?

Reply:
So the motivation for it existing is that the code has been compiled prior to debugging starting. Essentially the F5 "get me to debugging" workflow. I'd be concerned that having the users need to know that they need to run `tsc -w` to use it would make it more likely that they'd fail to use it successfully.

Reply:
Makes sense. Can `problemMatcher` be used to glob onto an existing `pnpm build` in a CLI?

Reply:
It's definitely possible to write your own problem matcher:

https://code.visualstudio.com/docs/editor/tasks#_defining-a-problem-matcher

I've never done it and I'm not sure if the output of tsup lends itself to it or not. But I'm not quite sure what you're thinking?

Reply:
Like, if the user is running either `pnpm build` or `pnpm run build` in a terminal window, VS Code should know not to create a new terminal with the build task. It should know to re-use the existing terminal. 

...actually, yeah, doesn't look like that's a job for `problemMatcher`.

Reply:
I see.  I'm not sure if that is possible top be honest.  I rather suspect that even if it was it would be flaky.  Also it might well be brittle as it would presumably potentially break as tsup changes over time.

Reply:
Yeah... ok, I don't want to be a stick in the mud. And I'm all for trying new things 🙂. Let's add it in as-is.